docs/modin_xgboost.rst

```diff
@@ -0,0 +1,142 @@
+Distributed XGBoost on Modin (experimental)
+===========================================
+
+Modin provides own effective implementation of a distributed XGBoost machine learning 
+algorithm on Modin DataFrames. Please note that this feature is experimental and behavior or 
+interfaces could be changed.
+
+Install XGBoost on Modin
+------------------------
+
+Modin comes with all the dependencies exclude of ``xgboost`` package by default.
+Currently, the distributed XGBoost on Modin supports the Ray backend only, therefore, see
+the `installation page`_ for more information on installing Modin with the Ray backend.
+To install ``xgboost`` package you can use ``pip``:
+
+.. code-block:: bash
+
+  pip install xgboost
+
+
+Learning Interfaces
+-------------------
+
+Distributed XGBoost functionality is placed in ``modin.experimental.xgboost`` module.
+``modin.experimental.xgboost`` provides a xgboost-like API for ``train`` and ``predict`` functions.
+
+``train`` has all arguments of ``xgboost.train`` function exclude the `evals_result`
+parameter which is returned as part of function return value instead of argument.
+
+``predict`` is separate function unlike ``xgboost.Booster.predict`` which uses an additional argument
+``model``. ``model`` could be ``xgboost.Booster`` or output of ``modin.experimental.xgboost`` function.
+
+Both of function have additional parameters `nthread` and `evenly_data_distribution`.
+`nthread` sets number of threads to use per node in cluster.
+`evenly_data_distribution` sets rule of distribution data between nodes in cluster.
+
+
+Type of input data
```

Comment:
This will go away if the method level documentation is added here.

Reply:
The method level docs were added.